e2e/utils/TestUtils.ts

```diff
@@ -6,7 +6,7 @@ import os from 'os';
 import path from 'path';
 import util from 'util';
 
-import { ElectronApplication, expect } from '@playwright/test';
+import { expect, _electron, ElectronApplication, Page } from '@playwright/test';
```

Comment:
FWIW, `ElectronApplication` and `Page` can be changed to an `import type`, but that's mostly just moving deck chairs.

Reply:
If it's alright, I'd like to keep it as it is. The following link is convincing to me: https://github.com/microsoft/TypeScript/issues/39861